Sources/SBTUITestTunnelServer/private/SBTProxyURLProtocol.m

```diff
@@ -673,6 +673,8 @@ -(void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)dataT
             [self.client URLProtocol:self didLoadData:stubResponse.data];
             [self.client URLProtocolDidFinishLoading:self];
             
+            completionHandler(NSURLSessionResponseAllow); 
+            
             return;
```

Comment:
I believe the same result can be achieved by simply removing the return statement.

Reply:
Updated!